plugins/hls-module-name-plugin/hls-module-name-plugin.cabal

```
@@ -16,6 +16,7 @@ extra-source-files:
   LICENSE
   test/testdata/*.hs
   test/testdata/*.yaml
+  test/testdata/subdir/*.hs
```

what about leverage globs available in cabal spec 2.4?
```suggestion
  test/testdata/**/*.yaml
  test/testdata/**/*.hs
```